### DISCUSSION TITLE
CorsixTH 0.67-beta1

### DISCUSSION THREAD

**Discussion by TheCycoONE** [CorsixTH/CorsixTH#2359] [Team Posts]
@CorsixTH/packagers I would like to release CorsixTH 0.67-beta 1 on July 8th, in anticipation of a final release late July or August. I have tagged the release v0.67-beta1 and started a draft release page where any artifacts can be dropped (e.g. mac packages.)

I hope that gives you enough time. Thanks everyone!

**Comment by colinjmatt** [comment]
macOS app has been uploaded.

**Comment by tim77** [comment]
Built for Fedora Rawhide and Fedora 38 [Testing](https://bodhi.fedoraproject.org/updates/FEDORA-2023-9130c1c729) (no pushing to Stable this beta version).
